Makefile.work

```diff
@@ -48,7 +48,7 @@ endif
 # Check for j2cli availability
 J2_VER := $(shell j2 --version 2>&1 | grep j2cli | awk '{printf $$2}')
 ifeq ($(J2_VER),)
-$(error Install j2cli)
+$(error Please install j2 (sudo snap install j2))
```

Comment:
Why snap and not `pip install j2cli`?

Reply:
I tried apt-get install j2. And got prompt to install with snap install j2.

So pip install j2cli is the proper way?

Reply:
snap only works for ubuntu, pip is better.